src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md

```diff
@@ -141,21 +141,25 @@ const {a: myA, b} = obj; // == const myA = obj.a, b = obj.b;
 ```
 As you saw above, you can also destructure an object received as an argument to a function:
 ```js
-url: ({index, size}) => {	// == url: (props) => { const index = props.index, size = props.size;
+url: ({index, size}) => {
+	// == const {index, size} = props;
+	// == const index = props.index, size = props.size;
 	return `//loremflickr.com/${size}/${size}/kitten?random=${index}`;
 }
 ```
+`index` prop is the index of kitten from [`Repeater` in `List` component](./#repeater). The function argument is a object which contains props of `Kitten` component, so `index` and `size` props in the function argument is destructured to the local variables named `index` and `size`.
+
 Destructuring is particularly useful in React because your render methods and computed properties will receive their input as a property object, which you can deconstruct into the relevant properties.
 
 ### Template Literals
 
-[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) ([spec](https://262.ecma-international.org/6.0/#sec-template-literals)) are a new string type in ES6 that support multi-line strings, expression interpolation, and tagged templates.
+[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) ([spec](https://262.ecma-international.org/6.0/#sec-template-literals)) are a string type in JavaScript that support multi-line strings, expression interpolation, and tagged templates.
```

Comment:
Actually, `template literals` are NOT strings. Commonly called `template strings` due to its common usage but it's literal. Let's find more exact description.

Reply:
I changed 'a string type' to 'literals' by referring to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).